### ISSUE TITLE
About the availability of the pretraining labels and data

### ISSUE THREAD

**Issue by Thomaswbt** [hspark1212/MOFTransformer#154]
Hi!

Thanks for your great work! I am currently interested in replicating the pretraining phase of MOFTransformer/PMTransformer. I wonder if there is possibility to release the pretraining labels("mtp", "vfp" and "moc") for better reproducibility of your work? As for the pretraining data, I notice the cif files are released, but is it possible to also release the preprocessed grid and graph data?

By the way, I am also curious about how you curate the "moc" labels? Can you elaborate on how you label the atoms as either metal nodes or organic linkers?

Thanks a lot in advance!

**Comment by Thomaswbt** [comment]
Sure, thank you! My email is ***@***.***

Yeonghun Kang ***@***.***> 于2024年3月5日周二 07:01写道：

**Comment by Yeonghun1675** [comment]
Hi @Thomaswbt ,
I apologize for the late reply. It took me a long time to find the pre-train material. We can provide you with the MTP VFP, and the labels for the MOC. However, I don't think we can deliver the grid and graph data due to capacity issues (it's over 60 GB, so I don't think we can deliver it directly).

Also, your email address seems to be blocked, so it will be difficult for us to send it to that address. Could you please send it again? @hspark1212 will be send the email to you an email.

**Comment by Thomaswbt** [comment]
Thank you for your reply! My email is thomaswbt123 at gmail dot com. Also may I ask how long would it approximately take to calculate the grid and graph data given the cifs?

**Comment by hspark1212** [comment]
I've emailed you JSON files that contain labeling data for the pre-training of MOFTransformer.

Generating the grid and graph data could take pretty long time (roughly guess one month) using a single job. However, we managed to complete the process in just 3 days, by leveraging parallel computing with 128 cores,

**Comment by Thomaswbt** [comment]
Thanks for your sharing! Closing the issue.

**Comment by tyvanzou** [comment]
> Hi @Thomaswbt , I apologize for the late reply. It took me a long time to find the pre-train material. We can provide you with the MTP VFP, and the labels for the MOC. However, I don't think we can deliver the grid and graph data due to capacity issues (it's over 60 GB, so I don't think we can deliver it directly).
> 
> Also, your email address seems to be blocked, so it will be difficult for us to send it to that address. Could you please send it again? @hspark1212 will be send the email to you an email.

Hi!

Would you please also send me the labels json file(MTP, VFP, and MOC)? My email is 24110240123@m.fudan.edu.cn. Thanks very much!

**Comment by tyvanzou** [comment]
> Hi!
> 
> Thanks for your great work! I am currently interested in replicating the pretraining phase of MOFTransformer/PMTransformer. I wonder if there is possibility to release the pretraining labels("mtp", "vfp" and "moc") for better reproducibility of your work? As for the pretraining data, I notice the cif files are released, but is it possible to also release the preprocessed grid and graph data?
> 
> By the way, I am also curious about how you curate the "moc" labels? Can you elaborate on how you label the atoms as either metal nodes or organic linkers?
> 
> Thanks a lot in advance!

@Thomaswbt 
Hi!

I am trying to reproduce the result in the paper. I've emailed the authors to obtain the MOC, VFP and MTP labels but still no response. I was a little anxious because of the deadline for revising my paper. Could you please kindly send me the label data? My email is 24110240123@m.fudan.edu.cn or 20307130189@fudan.edu.cn. Thanks very much!

**Comment by Yeonghun1675** [comment]
Hi @tyvanzou ,
I have sent you an email with a json file attached! please check your email!

**Comment by tyvanzou** [comment]
> Hi @tyvanzou , I have sent you an email with a json file attached! please check your email!

Got it. Really THANKS for your kind sharing and your contribution to the Materials AI community!

**Comment by chenyuaner** [comment]
> Hi [@tyvanzou](https://github.com/tyvanzou) ,  你好， I have sent you an email with a json file attached! please check your email!我已经给你发了一封电子邮件，并附上了一个json文件！请检查您的电子邮件！

Hi @Yeonghun1675 ，i also want the json files of vfp、mtp and moc, could you send me a copy? This is my email: csy3311582@163.com
